src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -517,5 +517,31 @@ public static void clearPostHearingRequestFormatAndContentFields(SscsCaseData ca
             }
         }
     }
+
+    public static YesNo isSorRequestInTime(SscsDocument sorDocument, SscsDocument finalDecision) {
+        if (isNull(sorDocument) || isNull(finalDecision)) {
+            return null;
+        }
+
+        String decisionDateAdded = finalDecision.getValue().getDocumentDateAdded();
+        if (isNull(decisionDateAdded)) {
+            return null;
+        }
+        LocalDate decisionDate = LocalDate.parse(decisionDateAdded);
```

Comment:
Could this be moved just before implementation? It's not being used in the next if statement.